exercises/03-Polling-SCM/README.md

```diff
@@ -1,8 +1,96 @@
-**Creating a Freestyle Project**
+**Polling SCM for Changes**
 =====================================================
 
+Now that you have your build steps, it's time to learn how to automate running them!
 
+Like most CI build systems, Jenkins is intended to be used to trigger builds when new changes are found in Source Code Management (SCM).
 
-**Step 1: Setup a Freestyle Project**
+In this exercise, you'll setup a Git repository and poll it for changes. If any are made, it will trigger the jobs you made in the last exercise.
+
+**Step 1: Setup a Git Repository**
+-------------------------------------------
+
+First, you'll need to have a Git repository to make changes to. You're free to make one of your own, or clone the [Demo Repo from my account](https://github.com/devbyaccident/Demo_Repo).
+
+> Note: If you aren't comfortable using Git, please let the instructor know so they can walk through the steps required.
+
+**Step 2: Setup a Github Access Token**
 -------------------------------------------
 
+Unless you only want to poll public repositories, and only under Github's rate limit, you're going to need to authenticate your git calls. There are many ways to do this, but for demo purposes, you'll be setting up an simple one today that doesn't require installing an application in Github.
+
+While logged into your Github account, go to https://github.com/settings/tokens.
+
+From here, navigate to **Personal access tokens (classic)** and create a new token. The token should have the scopes below:
```

Comment:
There's two types of tokens to generate for personal access tokens (classic) 
<img width="342" alt="image" src="https://github.com/user-attachments/assets/07574f42-cab5-47c2-bf34-6b7fe9cb4278">